README.md

```diff
@@ -152,3 +152,36 @@ slack.postMessage('hello slack', (err) => {
     // err is an Error object containing a message as to what was not defined.
 })
 ```
+
+### Health Check
+
+Set up an HTTP endpoint to monitor application health. Returns 200 if health and 500 if not healthy.
+
+The module can be configured as follows:
+
+```
+HEALTHCHECK_HTTP_LISTEN_PORT: optional, the listen port for the endpoint. Defaults to 8999
+
+```
+
+```js
+// Basic listener
+healthCheck.initListener();
+```
+
+```js
+// Advanced listener using promise
+function isAppHealthy() {
+    return new Promise((resolve, reject) => {
+        const isDatabaseOk = checkDatabaseConnectivityOk();
```

Comment:
This is clear enough, so no need to update, but in reality this would probably look more like:
```
function is AppHealthy() {
    return Promise.all([
        checkDatabaseConnectivityOk(),
        checkCacheConnectivityOk(),
    ]);
}
```

which is async af